### PR TITLE
fixing Bundle.list example

### PR DESCRIPTION
**Pull request by eksperts**
fixing syntax in the example snippet

This PR contains:
- [ ] Changelog
- [x] Documentation
- [ ] Tests

### CODE REVIEW

**Comment by kusma** [comment]
@eksperts: dashes is not the right way of crossing things off the list. A checkbox is either checked or not. If you want to remove it, you can either remove the checkbox, or just ~strike-out~ the the label.

**Comment by eksperts** [comment]
@kusma fixed

File: Source/FuseJS/Bundle.uno [hunk 1]
```diff
@@ -85,7 +85,6 @@ namespace FuseJS
 				.then(function(list) {
 					//list is an array of paths, such as "assets/image.jpg"
 				})
-			}
```
Comment (kusma) [inline Source/FuseJS/Bundle.uno]: The whole indentation of the .then etc seems off to me. Wouldn't it be cleaner to do
```js
Bundle.list().then(function(list) {
	//list is an array of paths, such as "assets/image.jpg"
})
```
?

Comment (eksperts) [reply Source/FuseJS/Bundle.uno]: I agree that would look better. Wanted my fix to be as minimal as possible, but I can change it to what you suggest.

**Pull request merged**